src/app/utils/getNews.ts

```diff
@@ -2,11 +2,16 @@ import { schemaNewsResponse } from '@/models/schemes';
 import { NewsData, USER_MESSAGE } from '@/models/types';
 import axios from 'axios';
 import { z } from 'zod';
+import objectToQueryString from './objectToQueryString';
 
-const getNews = async (keyword: string): Promise<NewsData> => {
+const getNews = async (keyword: string, params: { [key: string]: string } = {}): Promise<NewsData> => {
```

Comment:
If you need to pass types to an object use build in TS generic `Record<string, string>`

Reply:
But in the current case why don’t you use `Selects` type here?